app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,23 @@
+
+import React, { useEffect } from "react"
+import { getGreeting } from "../store/greetingsStore"
+import { useDispatch, useSelector } from "react-redux"
+
+const Greeting = () => {
+    const dispatch = useDispatch()
+    const greet = useSelector(state => state.greeting)
+
+    useEffect(() => {
+        dispatch(getGreeting())
+      }, [])
+    
+    console.log(greet);
+
+
+    return (
+        <h2>{greet.message}</h2>
+        // <button>greet</button>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to remove any unused commented code in your app, it makes the base code messy and hard to read